{"name": "test_notin", "status": "failed", "statusDetails": {"message": "AssertionError: assert 'Hi' in 'Hello World！'", "trace": "self = <pytest_1.test_one.Test_Class_In object at 0x10a73fdc0>\n\n    def test_notin(self):\n        a = \"Hello World！\"\n        b = \"Hi\"\n        c = \"World\"\n>       assert b in a\nE       AssertionError: assert 'Hi' in 'Hello World！'\n\npytest_1/test_one.py:28: AssertionError"}, "start": 1731236205293, "stop": 1731236205293, "uuid": "19a18044-4bc0-4af6-b844-3f67548ffb6c", "historyId": "054ec90ac862387da42bcb05b380aca4", "testCaseId": "054ec90ac862387da42bcb05b380aca4", "fullName": "PycharmProjects.WebSelenium.pytest_1.test_one.Test_Class_In#test_notin", "labels": [{"name": "parentSuite", "value": "PycharmProjects.WebSelenium.pytest_1"}, {"name": "suite", "value": "test_one"}, {"name": "subSuite", "value": "Test_Class_In"}, {"name": "host", "value": "yanfd"}, {"name": "thread", "value": "90249-MainThread"}, {"name": "framework", "value": "pytest"}, {"name": "language", "value": "cpython3"}, {"name": "package", "value": "PycharmProjects.WebSelenium.pytest_1.test_one"}]}